queries.sql

```diff
@@ -5,4 +5,61 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+--UPDATE SPECIES COLUMN
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL OR species = '';
+SELECT * FROM animals;
+
+COMMIT;
+SELECT * FROM animals;
+
+--DELETE 2022 BORN
+
+BEGIN;
+
+DELETE FROM animals
+WHERE date_of_birth  > '2022-01-01';
+
+SAVEPOINT first_save;
+
+UPDATE animals
+SET weight_kg  = weight_kg  * -1;
+
+ROLLBACK TO SAVEPOINT first_save;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+COMMIT;
+SELECT * FROM animals;
+
+--RESPOND QUESTS
+
+SELECT COUNT(*) name FROM animals;
+
+SELECT COUNT(*) name FROM animals WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) name FROM animals;
+
+SELECT neutered, SUM(escape_attempts) AS total_escape_attempts
+FROM animals
+GROUP BY neutered;
+
+SELECT neutered, MIN(weight_kg ), MAX(weight_kg)
+FROM animals
+GROUP BY neutered;
+
+SELECT species, AVG(escape_attempts) AS average_escape_attempts
+FROM animals
+WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
+GROUP BY species;
```

Comment:
- [ ] The screenshot you attached for this query displays the wrong result. The correct result should be `3.0`. If you look at the data in the `animals` table, only `Blossom` and `Squirtle` are born between _**1990 to 2000**_. They both have an `escape_attempt` of `3`. Therefore, the result that should be returned from this query is `3`. Kindly attach the correct screenshot. 👍 😎